packages/expo-sqlite/src/ExpoSQLiteNext.ts

```diff
@@ -1,43 +1,2 @@
-import type { SQLiteOpenOptions } from './NativeDatabase';
-import type { DatabaseChangeEvent } from './SQLiteDatabase';
-
-export default {
-  NativeDatabase(
-    databaseName: string,
-    options?: SQLiteOpenOptions,
-    serializedData?: Uint8Array
-  ): void {
-    throw new Error('Unimplemented');
-  },
-
-  NativeStatement(): void {
-    throw new Error('Unimplemented');
-  },
-
-  async deleteDatabaseAsync(databaseName: string): Promise<void> {
-    throw new Error('Unimplemented');
-  },
-
-  deleteDatabaseSync(databaseName: string): void {
-    throw new Error('Unimplemented');
-  },
-
-  importAssetDatabaseAsync(
-    databaseName: string,
-    assetDatabasePath: string,
-    forceOverwrite: boolean
-  ): Promise<void> {
-    throw new Error('Unimplemented');
-  },
-
-  //#region EventEmitter implementations
-
-  addListener(eventName: string, listener: (event: DatabaseChangeEvent) => void) {
-    throw new Error('Unimplemented');
-  },
-  removeListeners() {
-    throw new Error('Unimplemented');
-  },
-
-  //#endregion
-};
+import { requireNativeModule } from 'expo-modules-core';
+export default requireNativeModule('ExpoSQLiteNext');
```

Comment:
Nice 👍 I would recommend importing `requireNativeModule` from `expo` now (public APIs).

```suggestion
import { requireNativeModule } from 'expo';
export default requireNativeModule('ExpoSQLiteNext');
```